src/Evolution/Systems/ScalarWave/TagsDeclarations.hpp

```diff
@@ -0,0 +1,21 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+namespace ScalarWave {
+struct Psi;
+struct Pi;
+template <size_t Dim>
+struct Phi;
+
+/// \brief Tags for the ScalarWave evolution system
+namespace Tags {
```

Comment:
Thanks for finally introducing the `ScalarWave::Tags` namespace :) We should move the field tags here.

Reply:
We'll move all the missing ones soon, see #1897 !